2016/02/04 20:26:32 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "validatePassword".' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:266
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(227): CComponent->__call('validatePasswor...', Array)
#1 D:\PHP\xampp\htdocs\yiitest\protected\components\UserIdentity.php(41): CActiveRecord->__call('validatePasswor...', Array)
#2 D:\PHP\xampp\htdocs\yiitest\protected\components\UserIdentity.php(41): User->validatePassword('test')
#3 D:\PHP\xampp\htdocs\yiitest\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#7 D:\PHP\xampp\htdocs\yiitest\protected\controllers\SiteController.php(94): CModel->validate()
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('login')
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yiitest/index.php?r=site/login
HTTP_REFERER=http://localhost/yiitest/index.php?r=site/login
---
2016/02/04 21:51:09 [error] [php] htmlspecialchars() expects parameter 1 to be string, object given (D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\helpers\CHtml.php:108)
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\helpers\CHtml.php(163): renderAttributes()
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\helpers\CHtml.php(2455): tag()
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\helpers\CHtml.php(1534): activeInputField()
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\widgets\CActiveForm.php(792): activeTextField()
#4 D:\PHP\xampp\htdocs\yiitest\protected\views\post\_form.php(36): CActiveForm->textField()
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require()
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): PostController->renderInternal()
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): PostController->renderFile()
#8 D:\PHP\xampp\htdocs\yiitest\protected\views\post\create.php(18): PostController->renderPartial()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require()
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): PostController->renderInternal()
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): PostController->renderFile()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): PostController->renderPartial()
#13 D:\PHP\xampp\htdocs\yiitest\protected\controllers\PostController.php(81): PostController->render()
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams()
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): PostController->runAction()
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CAccessControlFilter->filter()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): PostController->filterAccessControl()
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): PostController->runActionWithFilters()
#23 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): PostController->run()
#24 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController()
#25 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#26 D:\PHP\xampp\htdocs\yiitest\index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php?r=post/create
in D:\PHP\xampp\htdocs\yiitest\protected\views\post\_form.php (36)
in D:\PHP\xampp\htdocs\yiitest\protected\views\post\create.php (18)
in D:\PHP\xampp\htdocs\yiitest\protected\controllers\PostController.php (81)
2016/02/04 21:52:11 [error] [exception.CException] exception 'CException' with message 'Property "PostController.updateTime" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:173
Stack trace:
#0 D:\PHP\xampp\htdocs\yiitest\protected\controllers\PostController.php(73): CComponent->__set('updateTime', 1454619131)
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yiitest/index.php?r=post/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=post/create
---
2016/02/08 11:25:14 [error] [exception.CException] exception 'CException' with message 'Property "PostController.updateTime" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:173
Stack trace:
#0 D:\PHP\xampp\htdocs\yiitest\protected\controllers\PostController.php(73): CComponent->__set('updateTime', 1454927114)
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): PostController->actionCreate()
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('post/create')
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yiitest/index.php?r=post/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=post/create
---
2016/02/08 11:30:28 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('delete')
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('post/delete')
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yiitest/index.php?r=post/delete&id=1
HTTP_REFERER=http://localhost/yiitest/index.php?r=post/view&id=1
---
2016/02/08 12:42:40 [error] [php] Undefined variable: model (D:\PHP\xampp\htdocs\yiitest\protected\views\user\_view.php:23)
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CListView.php(291): UserController->renderFile()
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#2 unknown(0): CListView->renderSection()
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback()
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(149): CListView->renderContent()
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(181): CListView->run()
#6 D:\PHP\xampp\htdocs\yiitest\protected\views\user\index.php(20): UserController->widget()
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require()
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): UserController->renderInternal()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): UserController->renderFile()
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): UserController->renderPartial()
#11 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(132): UserController->render()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams()
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): UserController->runAction()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CAccessControlFilter->filter()
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): UserController->filterAccessControl()
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): UserController->runActionWithFilters()
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): UserController->run()
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController()
#23 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#24 D:\PHP\xampp\htdocs\yiitest\index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php?r=user
in D:\PHP\xampp\htdocs\yiitest\protected\views\user\_view.php (23)
in D:\PHP\xampp\htdocs\yiitest\protected\views\user\index.php (20)
in D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php (132)
2016/02/08 12:47:18 [error] [exception.CException] exception 'CException' with message 'Property "User.image" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:173
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(161): CComponent->__set('image', NULL)
#1 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(74): CActiveRecord->__set('image', NULL)
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yiitest/index.php?r=user/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=user/create
---
2016/02/08 13:20:53 [error] [php] Undefined variable: sectionnames (D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php:95)
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): UserController->runAction()
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): UserController->filterAccessControl()
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): UserController->run()
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 D:\PHP\xampp\htdocs\yiitest\index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php?r=user/create
in D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php (95)
in D:\PHP\xampp\htdocs\yiitest\index.php (13)
2016/02/08 13:21:18 [error] [exception.CException] exception 'CException' with message 'Property "User.create_time" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('create_time')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_view.php(23): CActiveRecord->__get('create_time')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(130): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, false)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CListView.php(291): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array)
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#6 [internal function]: CBaseListView->renderSection(Array)
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback('/{(\\w+)}/', Array, '{summary}\n{sort...')
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(149): CBaseListView->renderContent()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(181): CBaseListView->run()
#10 D:\PHP\xampp\htdocs\yiitest\protected\views\user\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#15 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(168): CController->render('index', Array)
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#24 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('')
#26 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#27 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#28 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/yiitest/index.php?r=user
HTTP_REFERER=http://localhost/yiitest/index.php?r=user/create
---
2016/02/08 14:37:13 [error] [exception.CException] exception 'CException' with message 'Property "User.create_time" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('create_time')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_form.php(44): CActiveRecord->__get('create_time')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#5 D:\PHP\xampp\htdocs\yiitest\protected\views\user\create.php(18): CController->renderPartial('_form', Array)
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(95): CController->render('create', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/yiitest/index.php?r=user/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=user
---
2016/02/08 14:44:33 [error] [exception.CException] exception 'CException' with message 'Property "User.create_time" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('create_time')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_view.php(23): CActiveRecord->__get('create_time')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(130): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, false)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CListView.php(291): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array)
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#6 [internal function]: CBaseListView->renderSection(Array)
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback('/{(\\w+)}/', Array, '{summary}\n{sort...')
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(149): CBaseListView->renderContent()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(181): CBaseListView->run()
#10 D:\PHP\xampp\htdocs\yiitest\protected\views\user\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#15 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(168): CController->render('index', Array)
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#24 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('')
#26 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#27 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#28 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/yiitest/index.php?r=user
HTTP_REFERER=http://localhost/yiitest/index.php?r=user/create
---
2016/02/08 14:44:34 [error] [exception.CException] exception 'CException' with message 'Property "User.create_time" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('create_time')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_form.php(44): CActiveRecord->__get('create_time')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#5 D:\PHP\xampp\htdocs\yiitest\protected\views\user\create.php(18): CController->renderPartial('_form', Array)
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(95): CController->render('create', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/yiitest/index.php?r=user/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=user
---
2016/02/08 14:45:52 [error] [exception.CException] exception 'CException' with message 'Property "User.create_time" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('create_time')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_view.php(23): CActiveRecord->__get('create_time')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(130): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, false)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CListView.php(291): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array)
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#6 [internal function]: CBaseListView->renderSection(Array)
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback('/{(\\w+)}/', Array, '{summary}\n{sort...')
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(149): CBaseListView->renderContent()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(181): CBaseListView->run()
#10 D:\PHP\xampp\htdocs\yiitest\protected\views\user\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#15 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(168): CController->render('index', Array)
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#24 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('')
#26 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#27 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#28 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/yiitest/index.php?r=user
HTTP_REFERER=http://localhost/yiitest/
---
2016/02/08 14:46:51 [error] [exception.CException] exception 'CException' with message 'Property "User.create_time" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('create_time')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_form.php(44): CActiveRecord->__get('create_time')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#5 D:\PHP\xampp\htdocs\yiitest\protected\views\user\create.php(18): CController->renderPartial('_form', Array)
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(95): CController->render('create', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/yiitest/index.php?r=user/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=user
---
2016/02/08 14:48:05 [error] [exception.CException] exception 'CException' with message 'Property "User.create_time" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('create_time')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_form.php(44): CActiveRecord->__get('create_time')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#5 D:\PHP\xampp\htdocs\yiitest\protected\views\user\create.php(18): CController->renderPartial('_form', Array)
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(95): CController->render('create', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/yiitest/index.php?r=user/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=user
---
2016/02/08 14:48:37 [error] [exception.CException] exception 'CException' with message 'Property "User.alias" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('alias')
#1 D:\PHP\xampp\htdocs\yiitest\protected\views\user\_form.php(44): CActiveRecord->__get('alias')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#5 D:\PHP\xampp\htdocs\yiitest\protected\views\user\create.php(18): CController->renderPartial('_form', Array)
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require('D:\\PHP\\xampp\\ht...')
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\PHP\\xampp\\ht...', Array, true)
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): CBaseController->renderFile('D:\\PHP\\xampp\\ht...', Array, true)
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(95): CController->render('create', Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/yiitest/index.php?r=user/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=user
---
2016/02/08 14:52:14 [error] [exception.CException] exception 'CException' with message 'Property "User.alias" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('alias')
#1 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(77): CActiveRecord->__get('alias')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionCreate()
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('create')
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/create')
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yiitest/index.php?r=user/create
HTTP_REFERER=http://localhost/yiitest/index.php?r=user/create
---
2016/02/08 16:40:17 [error] [exception.CException] exception 'CException' with message 'Property "User.alias" is not defined.' in D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CComponent.php:130
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\db\ar\CActiveRecord.php(145): CComponent->__get('alias')
#1 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(110): CActiveRecord->__get('alias')
#2 [internal function]: UserController->actionUpdate('7')
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('update')
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/update')
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/yiitest/index.php?r=user/update&id=7
HTTP_REFERER=http://localhost/yiitest/index.php?r=user/update&id=7
---
2016/02/09 13:24:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php:199
Stack trace:
#0 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(103): UserController->loadModel('10')
#1 [internal function]: UserController->actionUpdate('10')
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController('user/update')
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 D:\PHP\xampp\htdocs\yiitest\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yiitest/index.php?r=user/update&id=10
---
2016/02/09 13:27:32 [error] [php] Undefined variable: model (D:\PHP\xampp\htdocs\yiitest\protected\views\user\_view.php:9)
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CListView.php(291): UserController->renderFile()
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(181): CListView->renderItems()
#2 unknown(0): CListView->renderSection()
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(164): preg_replace_callback()
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\zii\widgets\CBaseListView.php(149): CListView->renderContent()
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(181): CListView->run()
#6 D:\PHP\xampp\htdocs\yiitest\protected\views\user\index.php(20): UserController->widget()
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(126): require()
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CBaseController.php(95): UserController->renderInternal()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(869): UserController->renderFile()
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(782): UserController->renderPartial()
#11 D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php(170): UserController->render()
#12 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(49): UserController->actionIndex()
#13 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams()
#14 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): UserController->runAction()
#15 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#16 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CAccessControlFilter->filter()
#17 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): UserController->filterAccessControl()
#18 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#19 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#20 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): UserController->runActionWithFilters()
#21 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): UserController->run()
#22 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController()
#23 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#24 D:\PHP\xampp\htdocs\yiitest\index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php?r=user/index
in D:\PHP\xampp\htdocs\yiitest\protected\views\user\_view.php (9)
in D:\PHP\xampp\htdocs\yiitest\protected\views\user\index.php (20)
in D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php (170)
2016/02/09 15:23:47 [error] [php] Array to string conversion (D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php:112)
Stack trace:
#0 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(133): UserController->runAction()
#3 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CInlineFilter.php(58): UserController->filterAccessControl()
#6 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(291): CFilterChain->run()
#8 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CController.php(265): UserController->runActionWithFilters()
#9 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(282): UserController->run()
#10 D:\PHP\xampp\htdocs\yii-1.1.17\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 D:\PHP\xampp\htdocs\yii-1.1.17\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 D:\PHP\xampp\htdocs\yiitest\index.php(13): CWebApplication->run()
REQUEST_URI=/yiitest/index.php?r=user/update&id=11
in D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php (112)
in D:\PHP\xampp\htdocs\yiitest\index.php (13)
2016/02/09 17:34:24 [warning] [application] Failed to set unsafe attribute "image" of "User".
in D:\PHP\xampp\htdocs\yiitest\protected\controllers\UserController.php (112)
in D:\PHP\xampp\htdocs\yiitest\index.php (13)
